serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/elma/ELMALookupService.java

```diff
@@ -41,4 +43,32 @@ public Endpoint lookup(String organisationNumber) throws EndpointUrlNotFound {
             throw new EndpointUrlNotFound(String.format("Failed lookup through ELMA of %s", organisationNumber), e);
         }
     }
+
+    public boolean identifierHasInnsynskravCapability(String identifier) {
+        try {
+            Endpoint ep = lookupClient.getEndpoint(ParticipantIdentifier.of(identifier),
+                    DocumentTypeIdentifier.of(props.getElmaDPEInnsyn().getDocumentTypeIdentifier()),
+                    ProcessIdentifier.of(props.getElmaDPEInnsyn().getProcessIdentifier()),
+                    transportProfile);
+            return !isNullOrEmpty(ep.getAddress().toString());
+        } catch (PeppolSecurityException e) {
+            throw new ServiceRegistryException(e);
+        } catch (LookupException | EndpointNotFoundException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)